perun/postprocess/regressogram/methods.py

```diff
@@ -12,7 +12,16 @@
 _REQUIRED_KEYS = ['bucket_method', 'statistic_function']
 
 
-def get_supported_methods():
+def get_nparam_methods():
```

Comment:
Suggestion: I see (continuation of previous comment), I would rename this to `get_supported_nparam_methods` ;)